src/Servers/Kestrel/shared/KnownHeaders.cs

```diff
@@ -196,6 +93,8 @@ public static string GeneratedFile()
                 "TE",
                 "Translate",
                 "User-Agent",
+                "DNT",
+                "Upgrade-Insecure-Requests"
```

Comment:
:+1: Any other headers we should consider adding to the common headers list while we're thinking about it? @Tratcher

Reply:
I'd like to add a few more from https://github.com/aspnet/AspNetCore/blob/master/src/Http/Headers/src/HeaderNames.cs including the http/2 ones in that list of `:authority`,`:method`,`:path`, etc

Which was also part of my motivation for halving the size of `StringValues` https://github.com/aspnet/Extensions/pull/1283; so adding more headers wouldn't be too egregious.

However, was also looking to do that in a follow up, rather than this PR

Reply:
Makes sense. We should probably add the http/2 headers separately.